python/pylibcudf/pylibcudf/libcudf/labeling.pxd

```diff
@@ -1,14 +1,14 @@
 # Copyright (c) 2021-2024, NVIDIA CORPORATION.
-
+from libcpp cimport bool
 from libcpp.memory cimport unique_ptr
 from pylibcudf.libcudf.column.column cimport column
 from pylibcudf.libcudf.column.column_view cimport column_view
 
 
 cdef extern from "cudf/labeling/label_bins.hpp" namespace "cudf" nogil:
-    ctypedef enum inclusive:
-        YES "cudf::inclusive::YES"
-        NO "cudf::inclusive::NO"
+    cpdef enum class inclusive(bool):
```

Comment:
Should this enum be exposed in the public API according to https://docs.rapids.ai/api/cudf/stable/developer_guide/pylibcudf/#cython-scoped-enums?

Reply:
Yes. Note also that the `cudf::inclusive` enum has an underlying type of `int`, not `bool`.

```suggestion
    cpdef enum class inclusive(int):
```

Arguably that should be changed on the C++ side, although if we do that, we should change the order of the definition. So that `bool(cudf::inclusive::YES)` is true.